src/raftstore/store/peer.rs

```diff
@@ -459,8 +444,14 @@ impl Peer {
         self.raft_group.mut_store()
     }
 
-    pub fn is_applying(&self) -> bool {
-        self.get_store().is_applying()
+    #[inline]
+    pub fn is_applying_snapshot(&self) -> bool {
+        self.get_store().is_applying_snapshot()
+    }
+
+    #[inline]
+    pub fn has_pending_snap(&self) -> bool {
```

Comment:
here we use snap, but above function use snapshot, do we need to unify the name?